src/components/Results.vue

```diff
@@ -129,14 +129,18 @@
     </modal>
 
   </div>
-  <button class="main-cta check" type="button" name="button">sprawdź swoje odpowiedzi</button>
+  <!-- <button class="main-cta check" type="button" name="button">sprawdź swoje odpowiedzi</button> -->
+  <button class="main-cta" type="button" name="button" v-on:click="goToShareResults()">Podziel się wynikiem</button>
 </div>
 </template>
 
 <script>
   import PieChart from '@/components/PieChart'
   import Modal from '@/components/Modal'
 
+  import { default as ResultStats } from '../data/result_stats';
+  import { default as ManipulationCategory } from '../data/manipulation_category';
```

Comment:
Tutaj podobnie - można bez klamerek
```suggestion
  import ResultStats from '../data/result_stats';
  import ManipulationCategory from '../data/manipulation_category';
```